### PR TITLE
cli help changes to indicate options that are not supported for F5 (yet)

### PR DESCRIPTION
**Pull request by rajatchopra**
@pecameron PTAL

/cc @knobunc

### CODE REVIEW

File: pkg/cmd/admin/router/router.go [hunk 1]
```diff
@@ -282,16 +282,16 @@ func NewCmdRouter(f *clientcmd.Factory, parentName, name string, out, errout io.
...
-	cmd.Flags().StringVar(&cfg.StatsUsername, "stats-user", cfg.StatsUsername, "If the underlying router implementation can provide statistics this is the requested username for auth.")
+	cmd.Flags().StringVar(&cfg.StatsPassword, "stats-password", cfg.StatsPassword, "If the underlying router implementation can provide statistics this is the requested password for auth.  If not set a password will be generated. Not available for external appliance based routers (e.g. F5)")
+	cmd.Flags().StringVar(&cfg.StatsUsername, "stats-user", cfg.StatsUsername, "If the underlying router implementation can provide statistics this is the requested username for auth. Not available for external appliance based routers (e.g. F5)")
 	cmd.Flags().BoolVar(&cfg.ExposeMetrics, "expose-metrics", cfg.ExposeMetrics, "If true, attempts to run an extra container in the pod to expose metrics - the image will either be set depending on the router implementation or provided with --metrics-image.")
```
Comment (smarterclayton) [inline pkg/cmd/admin/router/router.go]: Can you mark this and the one below it deprecated?  I forgot to do it when I added the others.

Comment (rajatchopra) [inline pkg/cmd/admin/router/router.go]: Not really deprecated, we still allow a side container to be run with a specified image, don't we? The 'netscaler' integration plans to use this.

Or did you mean it is useless for haproxy.

Comment (smarterclayton) [inline pkg/cmd/admin/router/router.go]: I think for haproxy it should not be used.

**Comment by imcsk8** [comment]
LGTM

**Comment by rajatchopra** [comment]
[merge]

**Comment by smarterclayton** [comment]
flake TestConcurrentBuildPodControllers flake [merge]

**Comment by smarterclayton** [comment]
[merge]

**Comment by knobunc** [comment]
[merge] last flaked provisioning AWS (https://ci.openshift.redhat.com/jenkins/job/merge_pull_request_origin/472/)

**Comment by knobunc** [comment]
[merge] ... the last one just stopped for no obvious reason.

**Pull request merged**